schedule/yast/maintenance/autoyast_create_hdd_maintenance_deregister_aarch64.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: autoyast_create_hdd_maintenance_aarch64
```

Comment:
It's no relationship with maintenance. Why not just add deregister based on https://openqa.suse.de/tests/11049292/settings/schedule/functional/autoyast_create_hdd/autoyast_create_hdd_textmode.yaml to create a yaml schedule file suitable for all archs? Then the file name will be autoyast_create_hdd_deregister_textmode.yaml  WDYT?